Modules/_tkinter.c

```diff
@@ -1203,9 +1209,9 @@ typedef struct Tkapp_CallEvent {
 } Tkapp_CallEvent;
 
 void
-Tkapp_CallDeallocArgs(Tcl_Obj** objv, Tcl_Obj** objStore, int objc)
+Tkapp_CallDeallocArgs(Tcl_Obj** objv, Tcl_Obj** objStore, Tcl_Size objc)
 {
-    int i;
+    Tcl_Size i;
     for (i = 0; i < objc; i++)
```

Comment:
Watch out for conflict with #112679. I wonder if this function should iterate backwards over `objv[]` since it is deallocating.